code/modules/mob/mob_movement.dm

```diff
@@ -116,6 +116,10 @@
 
 	if(!mob.Process_Spacemove(direct))
 		return FALSE
+
+	if(SEND_SIGNAL(mob, COMSIG_MOB_CLIENT_PRE_MOVE, n) & COMSIG_MOB_CLIENT_BLOCK_PRE_MOVE)
```

Comment:
actually, one last thing since we're touching this, would you mind renaming the n var to `new_loc` or something a little easier to understand at a glance?

Reply:
Yeah, can do boss! I'll sweep it up in the next hour or so.

Reply:
Updated.